<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_SdBueNodeSelectorWidget</class>
 <widget class="QWidget" name="C_SdBueNodeSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="pc_VerticalLayout" stretch="0,0,0,1,1">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>1</number>
     </property>
     <property name="topMargin">
      <number>1</number>
     </property>
     <property name="rightMargin">
      <number>1</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="pc_BueNodeSelectorHeaderWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>39</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>39</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,1">
        <property name="leftMargin">
         <number>14</number>
        </property>
        <item>
         <widget class="QLabel" name="pc_LabelIcon">
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../../application.qrc">:/images/system_definition/IconNode.svg</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_BueNodeSelectorTitleLabel">
          <property name="text">
           <string>Nodes</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="pc_HorizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>294</width>
            <height>18</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="pc_VerticalSpacerTop">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="stw::opensyde_gui_elements::C_OgeLabGroupItem" name="pc_LabelNoNodes">
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="stw::opensyde_gui::C_SdBueNodeSelectorCheckBoxListWidget" name="pc_NodeSelectorListWidget" native="true"/>
     </item>
     <item>
      <spacer name="pc_VerticalSpacerBottom">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>stw::opensyde_gui_elements::C_OgeLabGroupItem</class>
   <extends>QLabel</extends>
   <header>C_OgeLabGroupItem.hpp</header>
  </customwidget>
  <customwidget>
   <class>stw::opensyde_gui::C_SdBueNodeSelectorCheckBoxListWidget</class>
   <extends>QWidget</extends>
   <header>C_SdBueNodeSelectorCheckBoxListWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../application.qrc"/>
 </resources>
 <connections/>
</ui>
